src/DeployPage.js

```diff
@@ -656,7 +663,7 @@ class Culprits extends React.PureComponent {
                   <span className="on-prefix">On</span> {group.name}
                 </h4>
                 {group.users.map(([role, user]) => (
-                  <div key={`${role}:${user}`} className="media">
+                  <div key={`${role}:${user.login}`} className="media">
```

Comment:
That `user` object doesn't have a `toString` so it would become (assume `role` is "Commenter") `'Commenter:[object object]'` which would be repeated.